Bender.yml

```diff
@@ -0,0 +1,31 @@
+package: 
+  name: fpu_interco
+  authors: 
+    - "Stefan Mach <smach@iis.ee.ethz.ch>"
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Andreas Kurth <akurth@iis.ee.ethz.ch>"
+    - "Francesco Conti <f.conti@unibo.it>"
+
+dependencies:
+  fpnew: { git: "git@github.com:pulp-platform/fpnew.git", version: 0.6.3 }
+  riscv: {path: "../riscv"} # Reference to be updated to git
```

Comment:
```suggestion
  riscv: { git: "git@github.com:pulp-platform/riscv.git", version: master } # There don't seem to be recent releases
```

Reply:
Setting riscv to master will make this incompatible with other dependencies of pulp, as the up-to-date master refers to cv32e40p. There are some changes required to the [Bender.yml file in riscv](https://github.com/micprog/cv32e40p/commit/29a838f6e2811b12220cb13bb02b37bd8ccd012f), compared to the state included in pulp (tag pulpissimo-v3.4.0). I believe there is work being done to convert to cv32e40p, I haven't looked into it yet. I will temporarily set the dependency to a forked repo, I'm open for ideas on proper integration.